.github/workflows/build.yml

```
@@ -2,7 +2,7 @@
 
 on:
   schedule:
-  - cron: "0 */6 * * *"
+  - cron: "0 */1 * * *"
```

Given GHA cron syntax complies with POSIX cron, it should better be simplified to this:
```suggestion
  - cron: "0 * * * *"
```